packit_service/service/api/copr_builds.py

```diff
@@ -37,6 +37,13 @@ def get(self):
             build_info = CoprBuildTargetModel.get_by_build_id(build.build_id, None)
             if build_info.status == BuildStatus.waiting_for_srpm:
                 continue
+            if (
+                build_info.status == BuildStatus.failure
+                and not build_info.build_start_time
+                and not build_info.build_logs_url
+            ):
+                # SRPM build failed, it doesn't make sense to list this build
```

Comment:
I would argue that this makes debugging of the failed SRPM builds harder :eyes: even though there is a status; I'd probably prefer fixing this on the dashboard side?

Reply:
Failed SRPM builds can be listed by `/api/srpm-builds` and have their own tab in the dashboard.
What do you mean by fixing it on dashboard side? Doing the filtering there?

Reply:
Oh, I get it now